### PR TITLE
Update using-ldap.md

### PR DESCRIPTION
**Pull request by adamrr724**
Changing enable LDAP Sync header to `##` so that it shows up in quick links along the right side of the article.

### Why: Attempting to add this to the quick links on the right side of the article by updating the header type. 

Closes: No issue created. 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Review by trip67 — COMMENTED** [review]
.

**Comment by cmwilson21** [comment]
👋 @adamrr724 Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by lecoursen — COMMENTED** [review]
Thanks @adamrr724! While reviewing this change on preview, it occurred to me that all these headers should actually be H2s. Would you mind updating the rest while we're here? 🙏🏻 Thanks!

File: content/admin/identity-and-access-management/using-ldap-for-enterprise-iam/using-ldap.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ When this option is selected, the certificate is validated to make sure:
 - The certificate is not expired.
 - The certificate is signed by a trusted certificate authority (CA).
 
-### Enabling LDAP Sync
```
Comment (jofmyg-jo) [inline content/admin/identity-and-access-management/using-ldap-for-enterprise-iam/using-ldap.md:112]:
```suggestion
### Enabling LDAP Sync
```

**Comment by nguyenalex836** [comment]
Hi @adamrr724! 👋 Just wanted to check if you saw @lecoursen's last comment regarding the H2s. If there's some context I'm missing, and this PR should just be closed out, let me know! 💛

**Review by adamrr724 — COMMENTED** [review]
Change to H2

**Comment by nguyenalex836** [comment]
Thank you! I'll update the branch and get this merged in 💛

**Comment by nguyenalex836** [comment]
@lecoursen Hi Laura! Would you be up for giving this PR a final review now that @adamrr724 has updated to H2s?

**Review by lecoursen — APPROVED** [review]
Looks great, thanks! ✨

**Pull request merged**